### DISCUSSION TITLE
Wrong password ssh connection

### DISCUSSION THREAD

**Discussion by Pavdi07** [alienatedsec/yi-hack-v5#282] [Q&A]
Hello,
I am new here and I would like ask your help. I have installed successfully yi hack v5 firmware on my yi outdoor 1080 camera. I can access to web ui on my ip:8080. I am trying to connect to my camera via ssh, but I get answer "Wrong password". I use right password and because I am noob, I dont know, if I am missing something important. Can someone give me an advice, whats wrong? I attached my printscreen. Thank you
![ssh](https://github.com/alienatedsec/yi-hack-v5/assets/121542765/d57ecfa8-d6c9-4e28-8ab4-32ca92d90021)
![home](https://github.com/alienatedsec/yi-hack-v5/assets/121542765/02764952-2e46-4f3e-8c76-3f84d5744715)

**Comment by alienatedsec** [comment]
@Pavdi07 Please select SCP when connecting using SSH

![image](https://github.com/alienatedsec/yi-hack-v5/assets/73167064/d286b75b-3930-43a2-93fd-75f889a8d466)

**Reply by Pavdi07** [reply]
same result, "Wrong password".

I think, I have larger problem, because my mqtt probably doesnt work too. I am using MQTT, Node red... first time, so I am not sure. But I received in Node Red "motin_detection" from camera earlier and now dont receive anything. Maybe I break something.

Can "Reset to YI-HACK-V5 defaults" help me?

**Reply by alienatedsec** [reply]
> Can "Reset to YI-HACK-V5 defaults" help me?

You can in the `service` section of the WebUI.
